src/index.js

```diff
@@ -1,14 +1,57 @@
-import _ from 'lodash';
 import './style.css';
+import '@fortawesome/fontawesome-free/js/fontawesome';
+import '@fortawesome/fontawesome-free/js/solid';
+import '@fortawesome/fontawesome-free/js/regular';
+import '@fortawesome/fontawesome-free/js/brands';
 
-function component() {
-  const element = document.createElement('div');
+class Task {
+  constructor(description, index) {
+    this.description = description;
+    this.index = index;
+    this.checked = false;
+  }
+}
+
+const todoInput = document.querySelector('.todo-input');
+const todoAdd = document.querySelector('.todo-add');
+const todoList = document.querySelector('.todo-list');
+const task1 = new Task('hello', 0);
+const task2 = new Task('hello', 1);
+const taskArr = [task1, task2];
+
+function addToUI(task) {
+  const todoDiv = document.createElement('div');
```

Comment:
- [ ] As per the learning objective we have to follow the guidelines of ES6 syntax. Kindly try to use the arrow function.

Reply:
This issue is well resolved! ✔️